cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -149,9 +150,11 @@ def test_message_catalog_translations(self):
         with mock.patch('gettext.translation', return_value=_translator(domain='text', localedir=localedir,
                                                                         languages=[get_language()])):
             i18n_service = self.get_module_i18n_service(self.descriptor)
-            self.assertEqual(i18n_service.ugettext('Hello'), 'Hello')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'fr-hello-world')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'es-hello-world')
+            self.assertEqual(i18n_service.gettext('Hello') if six.PY3 else i18n_service.ugettext('Hello'), 'Hello')
```

Comment:
We already created a utility function to do this, see [openedx/core/lib/edx_six.py](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/edx_six.py) - `get_gettext()`.